### PR TITLE
Add Question Service

### PR DESCRIPTION
**Pull request by blimyj**

### CODE REVIEW

File: question-service/api-routes.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+// router.route('/randomquestion')
+//     .post(questionController.randomQuestion);
+
+
+// router.route('/contacts/:contact_id')
+//     .get(questionController.view)
+//     .patch(questionController.update)
+//     .put(questionController.update)
+//     .delete(questionController.delete);
```
Comment (howardwhw2) [inline question-service/api-routes.js:35]: Do you still need this code?

File: question-service/question-service-mongodb/seed-mongodb/print-questions.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+//TODO: Fix having to Ctrl+C to terminate this program
+
+
+// for (let doc = await cursor.next(); doc != null; doc = await cursor.next()) {
+//   console.log("Question:\n")
+//   console.log(doc)
+//   console.log(!doc)
+//   console.log("End")
+// }
```
Comment (howardwhw2) [inline question-service/question-service-mongodb/seed-mongodb/print-questions.js:27]: Do you still need this code?

**Review by howardwhw2 — APPROVED** [review]
LGTM

File: question-service/question-service-mongodb/seed-mongodb/print-questions.js (hunk 2 above)
Comment (blimyj) [reply question-service/question-service-mongodb/seed-mongodb/print-questions.js]: Removed.

File: question-service/api-routes.js (hunk 1 above)
Comment (blimyj) [reply question-service/api-routes.js]: Removed.

**Pull request merged**